tests/integration/tests/conftest.py

```diff
@@ -37,8 +37,7 @@ def _generate_inspection_report(h: harness.Harness, instance_id: str):
     )
 
     (inspection_path / instance_id).mkdir(parents=True, exist_ok=True)
-    with open(inspection_path / instance_id / "inspection_report_logs.txt", "w") as f:
-        f.write(result.stdout)
+    Path(inspection_path / instance_id / "inspection_report_logs.txt", "w").write_text(result.stdout)
```

Comment:
you probably do not want the `"w"` there, also no need to add `Path()`, as `<path> / "string"` is a Path object already
```suggestion
    (inspection_path / instance_id / "inspection_report_logs.txt").write_text(result.stdout)
```